src/main/c/Makefile17

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/make -f
+#
+# Makefile for C code
```

Comment:
I don't believe we have build on Java17 currently (we test Java17 against code compiled by Java 8) so I'm not sure we would be able to test this.